### DISCUSSION TITLE
Icon rotation on rotated map: flat/billboard mode

### DISCUSSION THREAD

**Discussion by moving-bits** [mapsforge/mapsforge#1562] [Q&A] [answered]
When rotating a map, all symbols (markers) stay upright currently in Mapsforge, which is what VTM calls "billboard mode" in `MarkerSymbol`.
Sometimes I would like to have the symbol rotate with the map (`mBillboard = false` in VTM) - how can this be achieved with Mapsforge map? (Mapsforge's `Marker` class seems to not have a property for this.)

**Comment by devemux86** [comment, marked as answer]
@moving-bits This is easy to add as an option, see:
- #1563

**Reply by moving-bits** [reply]
Works like a charm, thanks for the quick fix!
